cdc/puller/sorter/unified_sorter.go

```diff
@@ -141,6 +166,16 @@ func (s *UnifiedSorter) Output() <-chan *model.PolymorphicEvent {
 	return s.outputCh
 }
 
+// RunWorkerPool runs the worker pool used by the heapSorters
+// It **must** be running for Unified Sorter to work.
+func RunWorkerPool(ctx context.Context) error {
```

Comment:
`heapSorterPool`/`heapSorterIOPool` is singleton. But `RunWorkerPool` could be called multiple times if cdc server suicides and recovers. Is this behavior safe?

Reply:
As long as for any given moment, there is at most one goroutine running `RunWorkerPool`, it will be fine. Cancelling the ctx passed to `RunWorkerPool` does not destroy any internal data, so as long as `RunWorkerPool` eventually runs, all the associated operations will be run.